src/Microsoft.Identity.Web/WebApiExtensions/MicrosoftIdentityWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -166,11 +166,13 @@ public static partial class MicrosoftIdentityWebApiAuthenticationBuilderExtensio
 
             // Change the authentication configuration to accommodate the Microsoft identity platform endpoint (v2.0).
             builder.Services.AddOptions<JwtBearerOptions>(jwtBearerScheme)
-                .Configure<IServiceProvider, IOptions<MicrosoftIdentityOptions>>((options, serviceProvider, microsoftIdentityOptions) =>
+                .Configure<IServiceProvider, IOptionsMonitor<MicrosoftIdentityOptions>>((options, serviceProvider, microsoftIdentityOptionsMonitor) =>
```

Comment:
Does this Configure method run twice for you?

It only runs once for me - for `Bearer` scheme. So whichever scheme (AAD or B2C) I set up as default `Bearer` is the only one that works.

Reply:
Interestingly @pmaytak, yes it only runs once. I was expecting it to be ran once for each options that we are adding. but it seems to only get fired once for the last schema added.